utam-compiler/src/main/java/utam/compiler/translator/ClassSerializer.java

```diff
@@ -95,6 +97,15 @@ public String toString() {
             .filter(method -> method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName()))
             .flatMap(method -> getMethodDeclaration(method).stream()).forEach(out::add);
     out.add(NEW_LINE);
+    source.getMethods().stream()
+        .filter(method -> (method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName())) && method.isElementMethod())
```

Comment:
second condition repeats line 97, so i d declare it as predicate and reuse
`Predicate<PageObjectMethod> isUsedMethod = method -> method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName())`

Reply:
After refactoring to the `addElementClassDeclarations` method, it makes more sense to make this a separate method, like `private boolean isUsedMethod(PageObjectMethod method)`, because it's now needed in multiple methods. So I've refactored that way.